samples/scenarios-monitoring/activity-log-alert.bicep

```diff
@@ -0,0 +1,39 @@
+param activityLogAlertName string = '${uniqueString(resourceGroup().id)}-alert'
+param actionGroupName string = 'testactiongroup'
+
+resource actionGroup 'Microsoft.Insights/actionGroups@2021-09-01' existing = {
```

Comment:
Should this use the 2021-09-01 API version?

Reply:
Sorry little confused, which API version should it use? 2021-09-01 is the latest version I can see

Reply:
Ignore me! I misread the version :D